lib/event_store_client/store_adapter/in_memory.rb

```diff
@@ -19,18 +19,20 @@ def append_to_stream(stream_name, events, expected_version: nil) # rubocop:disab
         end
       end
 
-      def read(stream_name, direction: 'forward', start: 0, count: per_page)
-        if direction == 'forward'
-          read_stream_forward(stream_name, start: start, count: count)
-        else
-          read_stream_backward(stream_name, start: start, count: count)
-        end
+      def read(stream_name, direction: 'forward', start: 0, count: per_page, resolve_links: nil)
+        read_stream_forward(stream_name, start: start, count: count) if direction == 'forward'
+
+        read_stream_backward(stream_name, start: start, count: count)
       end
 
       def delete_stream(stream_name, hard_delete: false) # rubocop:disable Lint/UnusedMethodArgument
         event_store.delete(stream_name)
       end
 
+      def link_to(stream_name, events)
+        raise NotImplementedError
```

Comment:
Don't you need that for testing?

It's easy - just add the events under new stream key - with the same ID and everything. Here we don't care about duplications, do we?

Reply:
I didn't even think that is so super easy to implement. So I just did it.